Frontend/android-native/app/src/main/java/in/juspay/mobility/CommonJsInterface.java

```diff
@@ -2254,6 +2254,8 @@ public void storeCallBackLocateOnMap(String callback) {
     @JavascriptInterface
     public void showMap(final String pureScriptId, boolean isEnableCurrentLocation, final String mapType, final float zoom, final String callback) {
         try {
+            View view = activity.findViewById(Integer.parseInt(pureScriptId));
```

Comment:
As this view we are not using These two lines can we replaced with the following
`if ((activity.findViewById(Integer.parseInt(pureScriptId)) == null) return;`